extension.js

```diff
@@ -66,8 +66,8 @@ class ForceQuitDialog extends ModalDialog.ModalDialog {
         super._init({ styleClass: 'force-quit-dialog' });
 
         const content = new Dialog.MessageDialogContent({
-            title: _('Quit applications'),
-            description: _("If an application doesn't respond for a while, select its name and click Quit Application."),
+            title: _('Force an Application to Quit'),
+            description: _("If an application doesn't respond for a while, select its name and click Force Quit. Any unsaved data will be lost."),
```

Comment:
“application” → “app”, and trying to avoid “click”. Maybe this is too much rewording, but I'd go for something like:

```suggestion
            description: _("If an app isn't responding or cannot otherwise be closed, select the app's name and choose Force Quit. Any unsaved data will be lost."),
```